app/javascript/react/containers/WizardShowContainer.js

```diff
@@ -30,7 +33,49 @@ class WizardShowContainer extends Component {
     .catch(error => console.error(`Error in fetch: ${error.message}`));
   }
 
+
+  addNewReview(formPayload) {
+
+    fetch('/api/v1/reviews', {
+      credentials: 'same-origin',
+      method: 'post',
+      body: JSON.stringify(formPayload),
+      headers: {
+        'Content-Type': 'application/json',
+        'X-Requested-With': 'XMLHttpRequest',
```

Comment:
Can you explain why you chose to include this please?

Reply:
This is part of passing the authenticity token to the api endpoint
added another header as well

Reply:
See my above comment on the api reviews controller. Once you have made that change, then you can remove those two additional headers